lib/mcb/commands/sites/geocode.rb

```diff
@@ -3,6 +3,7 @@
 usage "geocode [options] [<id>...]"
 option :s, "sleep", "time to sleep between each request", argument: :optional, default: 0.25, transform: method(:Float)
 option :b, "batch_size", "batch size", argument: :optional, default: 100, transform: method(:Integer)
+flag :f, "force", "Geocode even if the rest of the site has validation errors"
```

Comment:
_Suggestion_ "Geocode even if validation errors are present"

Reply:
I wanted to highlight that the validation errors where likely in the address, rather than the lat/long we are trying to update. I'm not really sure how to get it across clearly. If I read "Geocode even if validation errors are present" from cold I would probably assume that the errors were something created by the geo-coder rather than pre-existing errors. Thoughts?

Reply:
The validation errors may be address related but I also found there were some others that were violations of business rules e.g. duplicate names